lesson2/homework/helloworld/src/main/java/Animal.java

```diff
@@ -95,16 +98,18 @@ public void grow() {
 
 public class Farm {
 
+    private ArrayList<Animal> animals;
+
     public Farm(){
-        ArrayList animals = new ArrayList();
```

Comment:
This doesn't work because now your ArrayList is only accessible within your constructor!  We want `animals` to be accessible in the whole class, so we have to declare it as private outside any function.  Then we can set it to a `new ArrayList<>()` in our constructor, and it's accessible anywhere in this class!